docs/events/timeline-templates.asciidoc

```diff
@@ -6,11 +6,11 @@ alerts use the template when they are investigated in Timeline. This enables
 immediately viewing the alert's most interesting fields when you start an
 investigation.
 
-Templates can include two type of filters:
+Templates can include two types of filters:
 
 * *'Regular' filter*: Like other {kib} KQL filters, defines both the source
```

Comment:
I'm not aware of the original reason for italicizing `regular` (and other areas in this topic) but I've removed the italics around it. Also changed "ordinary" to "regular" in line 92 since I haven't seen any other references to an "ordinary" KQL filter in the Security or Kibana docs. Plus, the type of filter that's described in line 92 looks to be a regular filter--as opposed to a template filter.